<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" colspan="2" >
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QLineEdit" name="equationLineEdit" >
        <property name="maximumSize" >
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="font" >
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text" >
           <string>Constants:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addConstantsModelPointPushButton" >
          <property name="text" >
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearConstantsModelPushButton" >
          <property name="text" >
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1" >
       <widget class="QTableView" name="constantsTableView" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>256</width>
          <height>100</height>
         </size>
        </property>
        <property name="editTriggers" >
         <set>QAbstractItemView::DoubleClicked</set>
        </property>
        <property name="tabKeyNavigation" >
         <bool>true</bool>
        </property>
        <property name="showDropIndicator" stdset="0" >
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode" >
         <bool>false</bool>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2" >
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Independent Variable:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="independentVariableLineEdit" >
        <property name="maximumSize" >
         <size>
          <width>16666215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <widget class="QCheckBox" name="useRangeCheckBox" >
        <property name="text" >
         <string>Use The Following Range:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QDoubleSpinBox" name="lowerDoubleSpinBox" >
          <property name="minimum" >
           <double>-99.989999999999995</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="higherDoubleSpinBox" >
          <property name="minimum" >
           <double>-99.989999999999995</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="2" >
       <widget class="QPushButton" name="evaluatePushButton" >
        <property name="text" >
         <string>Evaluate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="3" >
     <widget class="QLabel" name="equationLabel" >
      <property name="font" >
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>Equation</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QTextBrowser" name="textBrowser" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2" >
     <widget class="GraphWidget" native="1" name="graphWidget" >
      <property name="acceptDrops" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuQuit" >
    <property name="title" >
     <string>Menu</string>
    </property>
    <addaction name="actionQuit" />
   </widget>
   <addaction name="menuQuit" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionQuit" >
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphWidget</class>
   <extends>QWidget</extends>
   <header>GraphWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
